examples/3d/3d_scene.rs

```diff
@@ -28,8 +28,14 @@ fn setup(
         ..Default::default()
     });
     // light
-    commands.spawn_bundle(PointLightBundle {
-        transform: Transform::from_xyz(4.0, 8.0, 4.0),
+    let transform = Transform::from_xyz(4.0, 8.0, 4.0).looking_at(Vec3::ZERO, Vec3::Y);
+    let mut directional_light = DirectionalLight::default();
+    directional_light.illuminance = 10000.0;
```

Comment:
```suggestion
```
`DirectionalLight::default()` already sets `illuminance` to 10000.0 so this line is redundant here.

Reply:
@NathanSWard The default is actually 100,000, so the override is quite a bit dimmer. :) I actually went ahead and added place separators to both values to make them a little more readable.

Reply:
Ahh yepp. 
Thanks for adding the `_` to the numbers, as that makes it much more readable :smile: